build_modules/lib/src/meta_module_clean_builder.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:convert';
+
+import 'package:graphs/graphs.dart';
+
+import 'package:build/build.dart';
+import 'meta_module.dart';
+import 'meta_module_builder.dart';
+import 'modules.dart';
+
+/// The extension for serialized clean meta module assets.
```

Comment:
I'm unsure about the name "clean"...

Maybe `.package_modules` and `.cross_package_modules` instead?

Reply:
Can the doc comment explain what "clean" means?

Reply:
Done.